napari/utils/misc.py

```diff
@@ -448,12 +450,12 @@ def pick_equality_operator(obj) -> Callable[[Any, Any], bool]:
     return operator.eq
 
 
-def dir_hash(path: Union[str, Path], include_paths=True, ignore_hidden=True):
+def dir_hash(
```

Comment:
Can you add a "parameters" section to the doc string here. (I know it was already there without one, but now is a good time to add it, thanks!!)